### PR TITLE
Add a Blogger importer for Social-Data

### PR DESCRIPTION
**Pull request by bwillard**
This creates a blogger imported for social data, storing photos in Drive. (This is related to #586)

There are also some related changes around G+ and Mastodon exports.

### CODE REVIEW

File: distributions/demo-server/build.gradle [hunk 1]
```diff
@@ -184,4 +184,15 @@ task installToolsAndDockerize {
     dependsOn ":client-rest:installTools", dockerize
 }
 
+configurations.all {
```
Comment (holachuy) [inline distributions/demo-server/build.gradle]: Is this related to this PR?

File: extensions/auth/portability-auth-google/src/main/java/org/datatransferproject/auth/google/GoogleOAuthConfig.java [hunk 2]
```diff
@@ -57,6 +55,7 @@ public String getTokenUrl() {
         .put("CONTACTS", ImmutableSet.of("https://www.googleapis.com/auth/contacts.readonly"))
         .put("MAIL", ImmutableSet.of("https://www.googleapis.com/auth/gmail.readonly"))
         .put("PHOTOS", ImmutableSet.of("https://www.googleapis.com/auth/photoslibrary.readonly"))
+        .put("SOCIAL-POSTS", ImmutableSet.of("https://www.googleapis.com/auth/plus.login"))
```
Comment (holachuy) [inline extensions/auth/portability-auth-google/src/main/java/org/datatransferproject/auth/google/GoogleOAuthConfig.java]: Does this work for both Blogger and Gplus? Or not used for both? Let's add a comment clarifying.

File: extensions/data-transfer/portability-data-transfer-google/build.gradle [hunk 3]
```diff
@@ -37,6 +38,10 @@ dependencies {
     compile("com.googlecode.ez-vcard:ez-vcard:${ezVcardVersion}")
     compile("com.google.apis:google-api-services-plus:${googlePlusVersion}")
 
+    // Needed for OpenSocial ActivityStreams
+    compile group: 'joda-time', name: 'joda-time', version: '2.3'
```
Comment (holachuy) [inline extensions/data-transfer/portability-data-transfer-google/build.gradle]: Do we want to make the version a constant in the global version file?

File: distributions/demo-server/build.gradle (hunk 1 above)
Comment (bwillard) [reply distributions/demo-server/build.gradle]: Only tangentially, you can't run DTP right now with out this change, so I included it in this PR

File: extensions/auth/portability-auth-google/src/main/java/org/datatransferproject/auth/google/GoogleOAuthConfig.java (hunk 2 above)
Comment (bwillard) [reply extensions/auth/portability-auth-google/src/main/java/org/datatransferproject/auth/google/GoogleOAuthConfig.java]: Just G+, added a comment.

File: extensions/data-transfer/portability-data-transfer-google/build.gradle (hunk 3 above)
Comment (bwillard) [reply extensions/data-transfer/portability-data-transfer-google/build.gradle]: No, Activity stream depends on this specific version of joda-time, So I don't think we want to generalize it.

**Comment by bwillard** [comment]
PTAL?

File: extensions/data-transfer/portability-data-transfer-google/build.gradle (hunk 3 above)
Comment (holachuy) [reply extensions/data-transfer/portability-data-transfer-google/build.gradle]: Ok, let's add a comment

**Comment by bwillard** [comment]
PTAL?
